test/deploy/kubernetes/provisioner-pods/deployment_data_node1.yaml

```diff
@@ -0,0 +1,100 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  name: data-node1
+  labels:
+    app: data-node1
+    id: aaa120a9e051d103c164f605615c32a4
+spec:
+  hostname: data-node1
+  restartPolicy: Never
+  nodeSelector:
+    node-name: node1
+  volumes:
+  - name: node-id
+    downwardAPI:
+      items:
+      - path: "id"
+        fieldRef:
+          fieldPath: metadata.labels['id']
+  - name: solution-config
+    configMap:
+      name: solution-config
+  - name: cortx-secret
+    secret:
+      secretName: cortx-secret
+  - name: shared-node1
+    persistentVolumeClaim:
+      claimName: shared-node1
+  - name: local-node1
+    persistentVolumeClaim:
+      claimName: local-node1
+  - name: sdb
+    persistentVolumeClaim:
+      claimName: dev-sdb-node1
+  - name: sdc
+    persistentVolumeClaim:
+      claimName: dev-sdc-node1
+  - name: sdd
+    persistentVolumeClaim:
+      claimName: dev-sdd-node1
+  - name: sde
+    persistentVolumeClaim:
+      claimName: dev-sde-node1
+  - name: sdf
+    persistentVolumeClaim:
+      claimName: dev-sdf-node1
+  - name: sdg
+    persistentVolumeClaim:
+      claimName: dev-sdg-node1
+  - name: sdh
+    persistentVolumeClaim:
+      claimName: dev-sdh-node1
+  - name: sdi
+    persistentVolumeClaim:
+      claimName: dev-sdi-node1
+  containers:
+  - name: cortx-data
+    image: ghcr.io/seagate/cortx-all:2.0.0-latest-custom-ci
+    imagePullPolicy: IfNotPresent
+    args:
+    - /bin/sh
+    - -c
+    - set -x;
+      echo -e "#!/bin/bash\necho $*" > /opt/seagate/cortx/utils/bin/utils_setup;
```

Comment:
Guess we can control this flow using environment variable to enable/disable mocking...